experimenter/experimenter/experiments/api/v5/types.py

```diff
@@ -468,6 +468,7 @@ class NimbusExperimentType(DjangoObjectType):
     is_localized = graphene.Boolean()
     is_rollout = graphene.Boolean()
     is_sticky = graphene.Boolean()
+    is_web = graphene.Boolean()
```

Comment:
This should use `NonNull` because every application has `is_web` defined.

```suggestion
    is_web = graphene.NonNull(graphene.Boolean())
```